Headers/DebugServer2/Types.h

```diff
@@ -301,6 +301,7 @@ struct MemoryRegionInfo {
   Address start;
   uint64_t length;
   uint32_t protection;
+  std::string name;
```

Comment:
For the sake of uniformity, you should add a call to `name.clear()` in `MemoryRegionInfo::clear()` and add an `std::string name_` argument to the second constructor.